src/nvim/macros.h

```diff
@@ -12,6 +12,13 @@
  * macros.h: macro definitions for often used code
  */
 
+#ifndef MIN
+# define MIN(X, Y) ((X) < (Y) ? (X) : (Y))
+#endif
+#ifndef MAX
+# define MAX(X, Y) ((X) > (Y) ? (X) : (Y))
+#endif
+
```

Comment:
@equalsraf I merged this in 2b39453912c0110eeae6acfbc67cae68c0a9cf73, so please drop this whence you get a chance.